### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by butonic** [owncloud/impersonate#4]

**Comment by PVince81** [comment]
For the frontend, need npm/karma/jasmine, see https://github.com/owncloud/activity/tree/master/tests/js

**Comment by felixboehm** [comment]
correct. I'd prefer to use acceptance tests (features), as we do with behat.

**Comment by felixboehm** [comment]
@SergioBertolinSG @davitol Please create a set of Scenarios
